packages/zowe-explorer/package.json

```diff
@@ -23,11 +23,6 @@
   "categories": [
     "Other"
   ],
-  "files": [
-    "out",
-    "README.md",
-    "resources"
-  ],
```

Comment:
Yes - as of the newest `@vscode/vsce` version, a warning is printed if you have both in your repo. It will only respect one or the other when packaging files.